ironfish/src/wallet/wallet.ts

```diff
@@ -1372,6 +1372,15 @@ export class Wallet {
     await this.removeAccount(account)
   }
 
+  async removeAccountByNameSynchronous(name: string): Promise<void> {
```

Comment:
Quick drive-by: it's confusing to me as a random reader that there is a function with the form `async ...Synchronous`. Maybe this is a pattern we use elsewhere, but I'm not sure what to do with this mentally.

Reply:
Good point. `remove...Blocking` or `remove...WithCleanup` would be more accurate. Changing.

Reply:
Actually, recent changes make this function entirely superfluous. Removing.